app/assets/stylesheets/application.scss

```diff
@@ -146,3 +158,24 @@ a:hover {
   text-align: inline;
 }
 
+#nameplate {
+  font-size: 1.6em;
+}
+
+.btn {
+     background:none;
+     border:none;
+     padding:0;
```

Comment:
Line should be indented 2 spaces, but was indented 5 spaces<br>Properties should be ordered background, border, color, cursor, font, padding<br>Colon after property should be followed by one space